Authenticators/Elkarte/elkarteauth.py

```diff
@@ -0,0 +1,887 @@
+#!/usr/bin/env python
+# -*- coding: utf-8
+
+# Elkarte modification Copyright (C) 2015 Antony Derham <antony@xydre.com>
```

Comment:
This copyright header is a bit non-standard. IANAL so I have no clue if that would have any implications legally but I would recommend we stick to how the BSD-3-Clause template from http://opensource.org/licenses/BSD-3-Clause suggests it. Meaning one line for each copyright holder followed by an "All right reserved.". I would suggest mentioning you derived it from SMF auth in the comment part below the license.